src/main/java/com/ericsson/ei/handlers/DownstreamExtractionHandler.java

```diff
@@ -39,21 +39,19 @@
     public void runExtraction(RulesObject rulesObject, String mergeId, String event, String aggregatedDbObject) {
         try {
             ObjectMapper mapper = new ObjectMapper();
-            log.debug("Start extraction of Aggregated Object:\n" + aggregatedDbObject + 
+            LOGGER.debug("Start extraction of Aggregated Object:\n" + aggregatedDbObject +
```

Comment:
Please redo this logging to use {} as well.

Reply:
Done.